specification/domainservices/resource-manager/Microsoft.AAD/stable/2020-01-01/domainservices.json

```diff
@@ -464,10 +464,22 @@
           "$ref": "#/definitions/LdapsSettings",
           "description": "Secure LDAP Settings"
         },
+        "resourceForestSettings": {
```

Comment:
These are some signfiicant changes to the definitions in this spec.  These were all missed in the spec and have been in the public API since inception?

Reply:
This has been around since the beginning, it was just missing.